volttron/platform/aip.py

```diff
@@ -242,8 +243,15 @@ def stop(self):
             stdout, stderr = process.communicate()
             _log.info("stopping agent: stdout {} stderr: {}".format(stdout, stderr))
             if process.returncode != 0:
```

Comment:
If the first call to `secure_stop_agent.sh` fails, we most likely are trying to call the script inside a Docker container. The current convention for storing the volttron source code in a container is currently "/code/volttron/". Thus we should check that location before finally giving up and throwing a Runtime error.